src/wp-includes/class-wp-image-editor-gd.php

```diff
@@ -323,7 +323,13 @@ public function crop( $src_x, $src_y, $src_w, $src_h, $dst_w = null, $dst_h = nu
 			$dst_h = $src_h;
 		}
 
-		$dst = wp_imagecreatetruecolor( $dst_w, $dst_h );
+		foreach( array( $src_x, $src_y, $src_w, $src_h, $dst_w, $dst_h ) as $value ) {
+			if ( ! is_numeric( $value ) || (int) $value <= 0 ) {
+				return new WP_Error( 'image_crop_error', __( 'Image crop failed.' ), $this->file );
+			}
+		}
+
+		$dst = wp_imagecreatetruecolor( (int) $dst_w, (int) $dst_h );
```

Comment:
It'd be good to update the `@return` for this to include `false`

Reply:
Updates the DocBlock in commit https://github.com/WordPress/wordpress-develop/pull/791/commits/7628731d63b368b2bbdbe0b986f6de033a43d4d4.